dotCMS/src/main/java/com/dotcms/cms/login/LoginServiceFactory.java

```diff
@@ -459,6 +467,25 @@ public User getLogInUser( HttpServletRequest req ){
             }
             return user;
         }
+
+        public User getLogInUser( ){
+            HttpServletRequest request = httpServletRequestThreadLocal.getRequest();
+            User user = null;
+
+            try {
+                if (request == null) {
+                    user = userAPI.getSystemUser();
```

Comment:
I don't think this logic is correct, we are returning the logged in user, why return the system user if no one is logged in?, I think this could cause future problems.